proto/cosmos/gov/v1/tx.proto

```diff
@@ -275,12 +281,38 @@ message MsgSubmitMultipleChoiceProposalResponse {
 message MsgUpdateMessageParams {
   option (cosmos.msg.v1.signer) = "authority";
 
-  string             authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
-  string             msg_url   = 2;
-  MessageBasedParams params    = 3;
+  // authority is the address that controls the module (defaults to x/gov unless overwritten).
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // msg_url is the url of the message to be updated.
+  string msg_url = 2;
+
+  // params are the new params to be set for the message.
+  MessageBasedParams params = 3;
 }
 
 // MsgUpdateMessageParamsResponse defines the Msg/UpdateMessageParams response type.
 //
 // Since: x/gov 1.0.0
-message MsgUpdateMessageParamsResponse {}
\ No newline at end of file
+message MsgUpdateMessageParamsResponse {}
+
+// MsgSudoExec defines a message to execute an inner message as the governance module.
+//
+// Since: x/gov 1.0.0
+message MsgSudoExec {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address that controls the module (defaults to x/gov unless overwritten).
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // msg is the arbitrary message to be executed.
+  google.protobuf.Any msg = 2 [(cosmos_proto.accepts_interface) = "cosmos.base.v1beta1.Msg"];
```

Comment:
how come 1 instead of many?

Reply:
On Linux if you use sudo, you sudo one command. If you want to sudo two commands you use it twice.

Same here, otherwise you could do within a proposal [sudo [msg 1, msg 2], sudo [msg 3]], instead of always [sudo msg1, sudo msg2, sudo msg3]

Reply:
makes sense, i like we are going with a standard like this

Reply:
Note that you can always do sudo foo && bar, but idk I preferred having one to not have multi msgs proposals that can accept multi msgs sudo.